concent_api/concent_api/settings/base.py

```diff
@@ -196,3 +204,10 @@
 # A global constant defining the length of the time window within which a requestor or a provider is supposed to
 # contact concent and send or receive a message as defined in the protocol.
 CONCENT_MESSAGING_TIME = 3600  # seconds
+
+# Loads git tag or commit id of the golem_messages package currently used.
+try:
+    repo = Repo(os.path.dirname(golem_messages.__file__), search_parent_directories=True)
+    GOLEM_MESSAGES_VERSION = repo.commit().name_rev
+except InvalidGitRepositoryError:
+    GOLEM_MESSAGES_VERSION = None
```

Comment:
1) I think that adding such a heavy piece of code to settings is not a good idea. Now even simply importing settings would require lots of dependencies: gitpython and git, golem-messages, all its cryptographic libraries and even OpenSSL. Sometimes we may want to import settings in scripts that should not have that much overhead.

    Also note that this information is not universally needed. We want to return the header from `concent-api` but there is also `gatekeeper` and later other features sharing the same code base but running somewhere else on the cluster (verification agent and eth wallet agent) that will never ever need it.

    I think this does not really work as a setting. I'd make it a function in `utils`. And then call that function during middleware initialization. Possibly, caching the result somewhere - but definitely not in settings.

2) I'm not sure if relying entirely on git is a good idea here. What if golem-messages is installed from a python package? Installing it directly from github is just a temporary arrangement.

Reply:
This will be not removed if I use golem_messages.__version__